### ISSUE TITLE
Rotation of a pdf annotation by a specified angle

### ISSUE THREAD

**Issue by annapras** [pymupdf/PyMuPDF#1460]
More of a question,  I believe..
Hello,

Mupdf is a great application unlike others

I have a requirement to overlay an annotation over existing text, have been using the free text annotate api for doing this passing a rect..I see we have options to rotate only multiples of 90.

My q is

A) is it possible to rotate by an arbitrary degree, for e.g 15degrees
B) is it possible to have annotation in a polygon, not necessarily a rectangle..

Thanks again for another great tool..

**Comment by JorjMcKie** [comment]
> A) is it possible to rotate by an arbitrary degree, for e.g 15degrees

No, Freetext annots can only rotate the text, and by mutiples of 90 degrees.

>  B) is it possible to have annotation in a polygon, not necessarily a rectangle..

No, free text can only appear in rectangle sized annotations.

There also is no way to implement these requests any time later.

You can of course insert other text and / or graphics in a permament, not annot-based way ...
